### PR TITLE
Update step 11 with high-order mesh images

### PR DESCRIPTION
**Pull request by agrayver**
I hope this is ok. The images I referred to are attached here. The actual description of the *write_high_order_mesh* is copied from step-65 since the method is essentially the same. 

Related to #7021

cycle_0
cycle_3

### CODE REVIEW

**Comment by drwells** [comment]
This looks good at a glance but I think you mixed in some commits from another PR.

**Comment by agrayver** [comment]
>  I think you mixed in some commits from another PR.

Now fixed.

File: examples/step-11/doc/results.dox [hunk 1]
```diff
@@ -42,3 +42,24 @@ of convergence but just to reduce the constant before the convergence
 order. On the other hand, using a cubic mapping only improves the
 result further insignificantly, except for the case of very coarse
 grids.
+
+We can also visualize the underlying meshes by using, for instance
```
Comment (kronbichler) [inline examples/step-11/doc/results.dox]:
```suggestion
We can also visualize the underlying meshes by using, for instance,
```

File: examples/step-11/doc/results.dox [hunk 2]
```diff
@@ -42,3 +42,24 @@ of convergence but just to reduce the constant before the convergence
...
+ParaView. The image below shows initial meshes for different mapping
+degrees:
+
+<img src="https://www.dealii.org/images/steps/developer/step-11-cycle_0.png" alt="">
+
+Clearly, the effect is most pronounced when we go from the linear to
+quadratic mapping. This is also reflected in the error values given
+in the table above. The effect of going from quadratic to cubic degree
+is less dramatic, but still tangible owning to a more accurace
```
Comment (kronbichler) [inline examples/step-11/doc/results.dox]:
```suggestion
is less dramatic, but still tangible owing to a more accurate
```

File: examples/step-11/doc/results.dox [hunk 3]
```diff
@@ -42,3 +42,24 @@ of convergence but just to reduce the constant before the convergence
...
+description of the circular boundary.
+
+Next, let's look at the meshes after three global refinements
+
+<img src="https://www.dealii.org/images/steps/developer/step-11-cycle_3.png" alt="">
+
+Here, the differences are much less visible, especially for higher order
+mappings. Indeed, at this refinement level the error values reported
+in the table are essentially identical between mappings of degree two
```
Comment (kronbichler) [inline examples/step-11/doc/results.dox]:
```suggestion
in the table are essentially identical between mappings of degrees two
```

**Comment by agrayver** [comment]
@kronbichler fixed, thanks.

File: examples/step-11/doc/results.dox [hunk 4]
```diff
@@ -42,3 +42,24 @@ of convergence but just to reduce the constant before the convergence
 order. On the other hand, using a cubic mapping only improves the
 result further insignificantly, except for the case of very coarse
 grids.
+
+We can also visualize the underlying meshes by using, for instance,
+ParaView. The image below shows initial meshes for different mapping
+degrees:
+
+<img src="https://www.dealii.org/images/steps/developer/step-11-cycle_0.png" alt="">
```
Comment (bangerth) [inline examples/step-11/doc/results.dox]: This is one of the two figures I need for #9720.

File: examples/step-11/doc/results.dox [hunk 5]
```diff
@@ -42,3 +42,24 @@ of convergence but just to reduce the constant before the convergence
...
+
+Clearly, the effect is most pronounced when we go from the linear to
+quadratic mapping. This is also reflected in the error values given
+in the table above. The effect of going from quadratic to cubic degree
+is less dramatic, but still tangible owing to a more accurate
+description of the circular boundary.
+
+Next, let's look at the meshes after three global refinements
+
+<img src="https://www.dealii.org/images/steps/developer/step-11-cycle_3.png" alt="">
```
Comment (bangerth) [inline examples/step-11/doc/results.dox]: This is the other.

**Pull request merged**